tests/unit/test_models_change.py

```
@@ -0,0 +1,392 @@
+        )
+        db_session.add(change1)
+        db_session.commit()
+
+        # Try to create second change with same change_number
+        change2 = ChangeModel(
+            change_number=change_data["change_number"],
+            description="Second change"
+        )
+        db_session.add(change2)
+
+        with pytest.raises(Exception):  # SQLAlchemy IntegrityError
```

Ruff (0.14.14)

[warning] Do not assert blind exception: `Exception`

(B017)